src/services

```diff
@@ -1069,3 +1069,20 @@ p({
     }
   """
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"XLIFFTranslationDAO",
+  "serve":true,
+  "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setAuthorize(false).setJournalName(\"XLIFFTranslations\").setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setOf(foam.i18n.XLIFFTranslationValue.getOwnClassInfo()).build();",
```

Comment:
please break this into multiple lines, using the """ format.  Helps with future commits.